pegen/pegen.h

```diff
@@ -110,18 +111,17 @@ void *CONSTRUCTOR(Parser *p, ...);
 #define EXTRA start_lineno, start_col_offset, end_lineno, end_col_offset, p->arena
 
 PyObject *new_identifier(Parser *, char *);
-jmp_buf error_marker;
 
-#define PROPAGATE_ERROR(errno) \
+#define PROPAGATE_ERROR(p, errno) \
```

Comment:
I'd say `errno` is not a great name for this macro argument (it's an infamous C stdlib global variabe :-).

Do we really need to pass a value to the macro? The macro could just always return -1 (or why not 1?).